web/app/components/document/sidebar/related-resources/list-item/resource.hbs

```diff
@@ -6,10 +6,7 @@
       <img src="/images/document.png" class="h-auto w-[18px]" />
     </div>
   {{else}}
-    <FlightIcon
-      @name={{get-link-icon this.url}}
-      class={{if (eq (get-link-icon this.url) "globe") "opacity-40"}}
-    />
+    <Favicon @url={{this.url}} />
```

Comment:
What's the UX if the URL is malformed/invalid?

Reply:
Google responds with a default image (globe icon, 16x16)